index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jay Melfah</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/a738fb17fd.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+
+</head>
+<body>
+  <!-- toolbar -->
+  <nav class="nav_bar">
+    <ul class="nav_menu">
+      <li><span class="name">Jay Melfah</span></li>
+      <li><span class="hamburger">☰</span></li>    
+    </ul>
+  </nav>   
+
+  <!-- Headline section -->
+  
+  <header class="container">
+    <div class="intro">
+      <p>Hey There.</p>
+      <p>I'm Jay</p>
+    </div>
+    <div class="job_description">
+      <p>I'm a Software Developer</p>
+    </div>
+    <div class="details">
+      <p>I can help you build a product, feature or website. 
+        Look through some of my work and experience! If you like what 
+       you see and have a project you need coded, don't hesitate to contact me. 
+      </p>  
+    </div>
+    <div class="social_links">
+      <ul class="links">
+        <li class="social"><a href="https://github.com/Jaymelfah"><i class="fab fa-github"></i></a></li>
+        <li class="social"><a href="linkedin.com/in/jeremiah-ekow-melfah-a4402a161"><i class="fa-brands fa-linkedin-in"></i></a></li>
+        <li class="social"><a href="https://angellist.com"><i class="fab fa-angellist"></i></a></li>
+        <li class="social"><a href="https://twitter.com/home"><i class="fab fa-twitter"></i></a></li>
+        <li class="social"><a href="https://medium.com/@jerasterix"><i class="fab fa-medium"></i></a></li>
+      </ul>
+    </div>
+    <div class="img">
+      <img src="./Images/Illustration.svg" alt="illustriation image"/>
+    </div>
```

Comment:
- [ ] You have done a great job adding and positioning the background image. However, this image is supposed to be added as a background image using CSS and not as an image with HTML. Kindly add it with CSS and not with HTML. 

Have a look at [this](https://www.w3schools.com/cssref/pr_background-image.asp) and [this](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image) awesome resources for examples of how you are supposed to implement this requested change.